### DISCUSSION TITLE
shortcut operation 快捷操作

### DISCUSSION THREAD

**Discussion by lx123code** [gitbito/bitoai#228] [Ideas]
Can you add a small icon to the method that can quickly click and select all the codes of the method and execute specific functions of Bitto? Such as interpreting code, generating comments, performance checking, etc.
Similar to Tabnine or TONGYI Lingma
![image](https://github.com/user-attachments/assets/967268b2-957f-4665-994e-b278998b1fc1)
Because you need to select all lines of code and then click the right mouse button every time to perform these operations, I would like to add a faster method, just as a suggestion, thank you

**Comment by nisha-bito** [comment]
hi @lx123code ,

We appreciate the feedback and will add this in our feature pipeline. 
We will get back once the functionality is available.

Thanks,
Nisha
